Library/Homebrew/cmd/link.rb

```diff
@@ -24,7 +24,15 @@ def link
     mode.dry_run = true if ARGV.dry_run?
 
     ARGV.kegs.each do |keg|
-      if keg.linked?
+      if keg.name == "openssl"
+        opoo <<-EOS.undent
+          Refusing to link: openssl
+          Linking keg-only OpenSSL means you may end up linking against the insecure,
+          deprecated system version while using the headers from the Homebrew version.
+          Instead, specify the full include/library paths to your compiler.
```

Comment:
The “[…] to your compiler.” reads kinda awkward to me. Maybe “[…] during the build.” works better?